tests/Feature/WebResultsRouteTest.php

```diff
@@ -118,8 +118,6 @@ public function test_results_route_retrieves_mismatches()
                 $itemMismatch->property_id,
                 $stringMismatchQid,
                 $stringMismatch->property_id,
-                // as well as item id of an item mismatch value
-                $itemMismatch->wikidata_value,
```

Comment:
Does this have to be removed? I don’t see any changes to the `MismatchFactory` in this PR, so the `wikidata_value` in this test should still always exist, right?